src/core/thread/mle.hpp

```diff
@@ -601,6 +601,19 @@ class Mle : public InstanceLocator
         return mRealmLocalAllThreadNodes.GetAddress();
     }
 
+#if OPENTHREAD_FTD && OPENTHREAD_CONFIG_BACKBONE_ROUTER_ENABLE
+    /**
+     * This method returns a reference to the All Network Backbone Routers Multicast Address.
+     *
+     * @returns A reference to the All Network Backbone Routers Multicast Address.
+     *
+     */
+    const Ip6::Address &GetNetworkAllBackboneRoutersAddress(void) const
```

Comment:
nit: keep the name order the same as spec? for example, `GetAllNetworkBackboneRoutersAddress()`

Reply:
hesitated in the naming... make sense to use the consistent name with the spec. will update.